<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>224</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Frustum</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="LabelRadius">
              <property name="text">
               <string>Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LabelNear">
              <property name="text">
               <string>Near</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LabelFar">
              <property name="text">
               <string>Far</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QSlider" name="SliderRadius">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderNear">
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderFar">
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="ValueRadius">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ValueNear">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ValueFar">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Distance cam√©ra</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Angle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QSlider" name="SliderDistCam">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="SliderAngle">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="ValueDistCam">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ValueAngle">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SliderRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ValueRadius</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderNear</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ValueNear</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderFar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ValueFar</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderDistCam</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ValueDistCam</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderAngle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ValueAngle</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
